src/bundle/Resources/public/js/scripts/admin.content.edit.js

```diff
@@ -147,6 +149,35 @@
                 });
         }, eZ.adminUiConfig.autosave.interval);
     }
+    const addClickWorkflowHandler = (event) => {
+        const applyButton = doc.querySelector('[name="ezplatform_content_forms_content_edit[workflow][apply]"]');
+
+        if (workflowEventListener) {
+            return;
+        }
+
+        applyButton.dataset.isFormValid = 0;
+        workflowEventListener = true;
```

Comment:
Actually, I wonder whether we need this additional condition because you cannot add the same event listener twice with `addEventListener`.
ref. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#other_notes

Also, to quote MDN: "calling removeEventListener() with arguments that do not identify any currently registered EventListener on the EventTarget has no effect."
ref. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/removeEventListener#notes

Because as far as I can see, you add the same `clickHandler` regardless the workflow button? I think that this logic can be simplified.
But maybe I don't understand something, or I overlooked something, please let me know. 🙂

Reply:
I didn't see Darek's comments before review.